.github/workflows/publish.yaml

```diff
@@ -332,12 +332,9 @@ jobs:
         if: github.event.inputs.publish_package != '1'
         run: npm install ${{steps.package-name.outputs.prop}}
 
-      - name: Create tiny .js file to require package
+      - name: Create tiny .mjs file to import package
         run: |
-          echo 'require("${{steps.package-name.outputs.prop}}")' > tmp.js
-
-      - name: Confirm to import errors when opening package
-        run: node tmp.js
```

Comment:
issue is coming from the Vue-axios package when importing. Seems to be a known issue with using it as a Vue plugin, was unable to get it working.